packages/apps/shopify-app-remix/src/server/authenticate/helpers/reject-bot-request.ts

```diff
@@ -2,11 +2,29 @@ import {isbot} from 'isbot';
 
 import type {BasicParams} from '../../types';
 
+const SHOPIFY_POS_USER_AGENT = /Shopify POS\/[0-9]+\.[0-9]+\.[0-9]+\/Android/;
+const SHOPIFY_MOBILE_ANDROID_USER_AGENT =
+  /Shopify Mobile\/Android\/[0-9]+\.[0-9]+\.[0-9]+/;
```

Comment:
Let's make it safer by just checking `Shopify POS` or `Shopify Mobile`. Even iOS userAgent works now but it doesn't guarantee in the future.